plugins/services/src/js/reducers/serviceForm/Containers.js

```diff
@@ -365,8 +370,9 @@ module.exports = {
       switch (type) {
         case ADD_ITEM:
           const name = `container-${newState.length + 1}`;
+          const newItem = Object.assign({}, DEFAULT_POD_CONTAINER, { name });
 
-          newState.push(Object.assign({}, DEFAULT_POD_CONTAINER, { name }));
+          newState.push(value || newItem);
```

Comment:
I addressed it for the containers file. But this is a pattern we used at all ADD_ITEM cases. I will fix the other files (reducers) in a separate PR.